DOc/docs/source/Options/user-story-5.rst

```diff
@@ -11,21 +11,21 @@ Keep track of your data quality
 User story
 ----------
 
-What is Lorem Ipsum?
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
-orem Ipsum has been the industry's standard dummy text ever since the 1500s, 
-when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
-It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. 
-It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software 
-like Aldus PageMaker including versions of Lorem Ipsum.
-
-Why do we use it?
-It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. 
-The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', 
-making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 
-'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose 
-(injected humour and the like).
- 
+When first starting to set up a data management organisation, there are a lot of different aspects to consider. 
+The data governance organisation needs to be set up.
+Business and technical data governance models need to be set up.
+And data quality needs to be monitored.
+
+Keeping an overview of the organisation's progress in all these activities can be a challenge.
+The Aurelius Atlas tool lets key stakeholders monitor the progress of the implementation, as well as the overall data governance quality.
+This allows the organisation to monitor its progress in applying data management.
+It also allows for better decision making on where to focus for the next improvement. 
```

Comment:
It allows you to make better decisions and clarifies what to focus on for a next  improvement